acceptance/tests/options/color.rb

```diff
@@ -0,0 +1,15 @@
+# This test is intended to ensure with --debug and --color, facter sends escape sequences to colorize the output
+test_name "C864545: --debug and --color command-line options should print DEBUG messages with color escape sequences" do
+  tag 'risk:high'
+
+  confine :except, :platform => 'windows' # On windows we don't get an escape sequence to detect to color change
```

Comment:
Nope I tried, over ssh the nothing that I did would get colorization to work.

When I logged into the console and facter from the command line, I was able to see the color change, but windows doesn't do rely on the use of an escape sequence being sent by the program that is running to change the color.